### PR TITLE
Implement onboarding for individual tests

### LINKED ISSUE #438: Implement onboarding for individual tests
idea-onboarding

### PR DESCRIPTION
**Pull request by lmorchard**
- Add ExperimentTourStep model & admin, expose content via API
- Add ExperimentTourDialogView UI to show tour steps
- Add ExperimentTourStep model fixture demo content
- Locale strings, tests, migrations
- Fix to bin/run-common.sh to run migrations without console input

Closes #438

### CODE REVIEW

**Comment by lmorchard** [comment]
Ugh. Looks like a few things needs tweaking in the client-side tests, but this could probably use a look whenever. Going to poke at the broken tests tomorrow.

File: testpilot/frontend/static-src/test/views/experiment-tour-dialog-view.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+  const view = new View({
+    model: model
+  });
+
+  view.render();
+
+  t.ok(view.query('.onboarding-modal'));
+
+  // const content = view.queryAll('.tour-content');
```
Comment (meandavejustice) [inline testpilot/frontend/static-src/test/views/experiment-tour-dialog-view.js]: commented code

Comment (lmorchard) [inline testpilot/frontend/static-src/test/views/experiment-tour-dialog-view.js]: Yeah, this is the test I mentioned that's not working yet and I'm still poking at

**Comment by lmorchard** [comment]
Just updated the commit with what looks like a passing frontend test.

**Comment by lmorchard** [comment]
Realized the frontend test didn't do a whole lot, so I went a little crazy making it run the view through its paces. Think I'm done futzing with this now. (Of course, I say that and immediately notice a linting failure. Fixed now)

**Comment by lmorchard** [comment]
Fixed a hiccup on Circle CI. (is that what the assignment was for?)

**Comment by meandavejustice** [comment]
@lmorchard Yes, thanks. I'll try and get this merged today

**Comment by meandavejustice** [comment]
That is why. thanks for fixing that up. I just pulled it down and it works great, good job @lmorchard :)

**Pull request merged**